src/pages/code-reviews-getting-started/index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Code reviews: getting started"
+date: 2018-05-03T12:07:50+03:00
+draft: false
+---
+
+What if I told you there is this really straightforward technique to make your project run smoother, faster, and with fewer bugs 🤔. Better yet, it’s completely free, requires only minimal tooling, and any developer can learn to use it in a matter of minutes 😳. This is how good code reviews are, and why you should use them.
+
+_Author: Thibaud Colas, Software Architect at Codemate Ltd._
+
+---
+
+## Why it’s good
+
+Developers tend to think of productivity in terms of “how fast can I write the code for this new feature”. We also tend to say “I write code” when describing our jobs. In practice though, professional software development is a matter of understanding the code that is already there: we simply spend much more time reading than writing. Productivity thus depends more on the overall code’s quality and maintainability, rather than the skills of individual contributors making changes.
+
+Joel Spolsky puts it neatly:
+
+> [It’s harder to read code than to write it](https://www.joelonsoftware.com/2000/04/06/things-you-should-never-do-part-i/).
+
+John Papa recently gave a great talk about what code readability means in practice:
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/56mETnrByBM" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+If you like numbers, another great article on the topic is [Code Reviews: Just Do It](https://blog.codinghorror.com/code-reviews-just-do-it/) by Jeff Atwood. You should definitely read it. I especially like its excerpts from [Code Complete](https://www.amazon.com/exec/obidos/ASIN/0735619670):
+
+> … software testing alone has limited effectiveness – the average defect detection rate is only 25 percent for unit testing, 35 percent for function testing, and 45 percent for integration testing. In contrast, the average effectiveness of design and code inspections are 55 and 60 percent. Case studies of review results have been impressive:
+>
+> In a software-maintenance organization, 55 percent of one-line maintenance changes were in error before code reviews were introduced. After reviews were introduced, only 2 percent of the changes were in error. When all changes were considered, 95 percent were correct the first time after reviews were introduced. Before reviews were introduced, under 20 percent were correct the first time.
+
```

Comment:
I feel like it would be good to have a recap from the quote here.

Some potential key points to drive in:
* automated and functional testing are good, but code reviews are a good complimentary step
* code reviews bumped the correctness of code (whatever that means) from under 20 percent to 95 percent; this is a huge improvement!
* this is maybe because it's easier to spot potential errors if there are multiple pairs of eyes looking at the same code
* statistics back all of this up